specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2021-11-01/registeredidentities.json

```diff
@@ -0,0 +1,372 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-11-01",
```

Comment:
Between preview version and GA version, there should be a date change by at least one day. I recommend to make the GA version something like 2021-12-01.

Reply:
As per below offline discussion with Vishnu Priya, signing off this.

All 3 services (Site recovery, backup and recovery service) should follow similar API version. We have already released site recovery API version (2021-11-01). The other two swagger PRs are based on previous versions and no change. Can you please make an exception on this? We will try to follow this guidance in future API version naming.